### ISSUE TITLE
QMetry +Perfecto

### ISSUE THREAD

**Issue by SurekhaG123** [qmetry/qaf#389]
Hi, 
I am having an issue with Qmetry when running in perfecto cloud for Mobile Web Browser.
I am not able to initialize the driver.

This is happening specifically when i try to initialize driver application.properties file

When try to trigger the driver from a common method i am able to success.

Can someone please help on this

**Comment by cjayswal** [comment]
This is not right place for Q & A. It is recommended to post questions with [details](http://sscce.org/) in [users group](https://groups.google.com/g/qaf-users) or in [stack-overflow](https://stackoverflow.com/questions/tagged/qaf) with qaf tag.
